guides/common/modules/ref_supported-puppet-versions-and-system-requirements.adoc

```diff
@@ -0,0 +1,19 @@
+[id="supported-puppet-versions-and-system-requirements_{context}"]
+= Supported Puppet Versions And System Requirements
+
+Before you begin with the Puppet integration, review the supported Puppet versions and system requirements.
+
+Supported Puppet Versions::
+{Project} supports the following Puppet versions:
+* Puppet 7
+ifndef::satellite[]
+* Puppet 6
+endif::[]
+
+System Requirements::
+Before you begin integrating Puppet with your {Project}, ensure that you meet the system requirements.
+For details, see https://puppet.com/docs/puppet/7/system_requirements.html[System Requirements for Puppet 7]
+ifndef::satellite[]
+or https://puppet.com/docs/puppet/6/system_requirements.html[System Requirements for Puppet 6]
```

Comment:
```suggestion
https://puppet.com/docs/puppet/6/system_requirements.html[System Requirements for Puppet 6]
```

Reply:
I would keep the word "or", because for Katello output, both links to Puppet 7 and Puppet 6 are going to be rendered and this makes it into a pretty sentence. The Puppet 6 link is never going to be present alone.